main.js

```diff
@@ -204,6 +237,8 @@ function HandleNode(svgNode, scaleX, scaleY, translateX, translateY) {
 
       // CIRCLE ----------------------------------------------------------------
       case 'circle':
+        output += GetPathColorCommandFromFill(preserveFill, svgElement);
```

Comment:
can we put this at the very start of the loop, and make GetPathColorCommandFromFill ignore <g>s? And since you don't really do anything fancy with the first param of that function, I'd prefer to write it as

```
if (preserveFill)
  output += ...
```